### ISSUE TITLE
image_tag with relative assets

### ISSUE THREAD

**Issue by danhalligan** [kubenstein/middleman-simple-thumbnailer#19]
I believe that when relative_assets is activated, the paths generated with image_tag are incorrect. In the following examples, `image_dir` is set to `figs`.

For example, without activating relative_assets, within a template the following code generates the following links. Note the first two calls use middleman-simple-thumbnailer and the second two don't. 

```
<%= image_tag "image.png", resize_to: '50x50' %>
<%= image_tag "/image.png", resize_to: '50x50' %>
<%= image_tag "image.png" %>
<%= image_tag "/image.png" %>
```
```
<img src="/figs/image.50x50.png" alt="Image.50x50" />
<img src="/image.50x50.png" alt="Image.50x50" />
<img src="/figs/image.png" alt="Image" />
<img src="/image.png" alt="Image" />
```
so far so good. However, with `activate :relative_assets` in build mode, we get the following:

```
<img src="figs/figs/image.50x50.png" alt="Image.50x50" />
<img src="figs/image.50x50.png" alt="Image.50x50" />
<img src="figs/image.png" alt="Image" />
<img src="image.png" alt="Image" />
```
It appears that relative_assets is already intercepting the call and adding the `image_dir` path, but `middleman_simple_thumbnailer` is perhaps adding it again?

**Comment by ymph** [comment]
Hello,
Thank you for the report.
I'm having a look at it.

regards,

ymh

**Comment by ymph** [comment]
Corrected in commit 9b5ac73b7deceb87ee81ef997a2eb6e5c090ec7f.

Thanks.

ymh

**Comment by danhalligan** [comment]
Thanks!
Dan
